src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -1433,12 +1434,34 @@ public XMPPError onSessionAccept(
         participant.setRTPDescription(answer);
         participant.addTransportFromJingle(answer);
 
+        MediaSourceMap sourcesAdvertised
+                = MediaSourceMap.getSourcesFromContent(answer);
+        MediaSourceGroupMap sourceGroupsAdvertised
+                = MediaSourceGroupMap.getSourceGroupsForContents(answer);
+        if (sourcesAdvertised.isEmpty())
```

Comment:
i know we talked about this, but there is a reason we *can't* just do this on the client, right?  i worry we'll introduce new issues by injecting fake ssrcs that the client isn't aware of.

Reply:
I don't know, I haven't looked at the client at all. I agree with your concern, but I think it also applies to some extent if we make client changes.

Reply:
the client would be a single source of truth with regards to its sources though.

Reply:
> the client would be a single source of truth with regards to its sources though.

True, at least temporarily, until ssrcs fade away completely.